.env.example

```diff
@@ -0,0 +1,2 @@
+PORT=
+MONGO_URI=
```

Comment:
Do we need this file?

Reply:
It is optional. It's just to provide an idea of what all variables env file takes.

Reply:
There is no need for this file if your not putting .env in gitignore imo

Reply:
Yes, it's needed. The `.env` file is never committed, instead, the `.env.example` file is committed with properties in use with default or no values. After cloning the app, we basically copy this file and add real parameters.

Reply:
.end added to .gitignore.